create/git.go

```diff
@@ -0,0 +1,66 @@
+package create
+
+import (
+	"fmt"
+	"os/exec"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/tarantool/cartridge-cli/common"
+	"github.com/tarantool/cartridge-cli/project"
+)
+
+const (
+	initialTagName   = "0.1.0"
+	initialCommitMsg = "Initial commit"
+)
+
+func initGitRepo(projectCtx *project.ProjectCtx) error {
+	// check that git is installed
+	if _, err := exec.LookPath("git"); err != nil {
+		return fmt.Errorf("git not found. " +
+			"You'll need to add the app to version control yourself later")
+	}
+
+	// init repo
+	initCmd := exec.Command("git", "init")
+	if err := common.RunCommand(initCmd, projectCtx.Path, false); err != nil {
+		if err != nil {
+			return fmt.Errorf("Failed to initialize git repo")
+		}
+	}
+
+	log.Debug("Initialized git repo")
+
+	// add files to index
+	addCmd := exec.Command("git", "add", "-A")
+	if err := common.RunCommand(addCmd, projectCtx.Path, false); err != nil {
+		if err != nil {
```

Comment:
You've already checked that error is not nil. At previous line.

Reply:
Yes, and then I copy pasted it about 6 times...